src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(String operation, String fruit, String quantity) {
+        this.operation = Operation.of(operation);
+        this.fruit = fruit;
+        this.quantity = Integer.parseInt(quantity);
```

Comment:
The constructor parameters should not be of type String for operation and quantity. It would be better to use the appropriate types directly, such as Operation for operation and int for quantity, to avoid unnecessary parsing inside the constructor.

Reply:
++